core/chains/evm/txmgr/orm.go

```diff
@@ -42,6 +42,57 @@ type orm struct {
 
 var _ ORM = (*orm)(nil)
 
+// Directly maps to database schema.
+// Do not modify type unless you
+// intend to modify the database schema
+type dbReceipt struct {
+	ID               int64
+	TxHash           common.Hash
+	BlockHash        common.Hash
+	BlockNumber      int64
+	TransactionIndex uint
+	Receipt          evmtypes.Receipt
```

Comment:
Not strictly related to this PR, but in previous benchmarks the serialization/deserialization of this object is very expensive. I've seen it take e.g. 10, 20% of _total_ chainlink execution time.

You might consider pulling a binary blob and doing lazy deserialization later at the point of use.

Reply:
Sounds like a great optimization but let's introduce that in a separate ticket